libs/core/kiln_ai/adapters/ml_model_list.py

```diff
@@ -700,6 +702,49 @@ class KilnModel(BaseModel):
             ),
         ],
     ),
+    # Claude Sonnet 4
+    KilnModel(
+        family=ModelFamily.claude,
+        name=ModelName.claude_sonnet_4,
+        friendly_name="Claude Sonnet 4",
+        providers=[
+            KilnModelProvider(
+                name=ModelProviderName.openrouter,
+                model_id="anthropic/claude-sonnet-4",
+                suggested_for_data_gen=True,
+                suggested_for_evals=True,
+            ),
+            KilnModelProvider(
+                name=ModelProviderName.anthropic,
+                model_id="claude-sonnet-4-20250514",
+                structured_output_mode=StructuredOutputMode.json_instructions,
+                suggested_for_data_gen=True,
+                suggested_for_evals=True,
+            ),
+        ],
+    ),
+    # Claude Opus 4
+    KilnModel(
+        family=ModelFamily.claude,
+        name=ModelName.claude_opus_4,
+        friendly_name="Claude Opus 4",
+        providers=[
+            KilnModelProvider(
+                name=ModelProviderName.openrouter,
+                model_id="anthropic/claude-opus-4",
+                structured_output_mode=StructuredOutputMode.json_instructions,
+                suggested_for_data_gen=True,
+                suggested_for_evals=True,
```

Comment:
Let's remove the suggested tags for Opus. We try to keep those only to a limited few (3/4 per category). For price reasons I'm not recommending o1/o1-pro, etc. I think Opus falls in that bucket (💸 ).